src/raft.h

```diff
@@ -1398,6 +1398,28 @@ RAFT_API void raft_uv_set_tracer(struct raft_io *io,
  */
 RAFT_API void raft_uv_set_auto_recovery(struct raft_io *io, bool flag);
 
+/**
+ * Values for the format version number that describes the segment and snapshot
+ * files.
+ */
+enum {
+	RAFT_UV_FORMAT_V1 = 1, /* Original */
+	RAFT_UV_FORMAT_V2,     /* Added local_data fields */
+	UV_FORMAT_NR
```

Comment:
Maybe `RAFT_UV_FORMAT_NR`? for consistency. Or is there are reason to have a distinct name?

Reply:
Oops, missed this with `sed`